runtime/sbf/syscalls.md

```diff
@@ -0,0 +1,130 @@
+# Syscalls
+
+Syscalls are VM hardcoded subroutines extending the functionality of the virtual machine.
+They generally belong to one of two categories:
+
+- *Runtime syscalls* provide access to external facilities (e.g. logging)
+- *Convenience syscalls* provide efficient replacements for functionality that could be implemented without syscalls (e.g. SHA-256 hashing)
+
+## Usage
+
+A syscall is invoked using a [`call` instruction](./isa.md#call) with `dst=0`, `src=0`, `off=0`, and `imm` set to the hash in reverse byte order.
```

Comment:
The way syscalls and normal function calls are differentiated was changed in SBFv2, see `config.static_syscalls`: https://github.com/solana-labs/rbpf/blob/e58129a441f306d024e8e7397428c0fb96df2224/src/interpreter.rs#L446